task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml

```
@@ -46,8 +42,29 @@ spec:
 
         When set to "true", skip verifying the TLS certs of the Central endpoint and registry.
       default: "false"
+    - name: rox_config_dir
+      type: string
+      description: |
+        Path to the roxtl config directory within the roxctl-config workspace (if machine to machine authentication is used).
+        Mutually exclusive with token_file.
+        The path should be prefixed with /roxctl-config .
```

nit: seems to be in line with what we do for other parameter values?
```suggestion
        The path should be prefixed with "/roxctl-config".
```